src/tests/JIT/Directed/Convert/value_numbering_checked_casts_of_constants.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <CLRTestPriority>0</CLRTestPriority>
```

Comment:
They should be moved to pri1, since there are a lot of exceptions being thrown / caught. On my Ivy Bridge they take ~3 seconds (under Checked runtime).

Fixed.